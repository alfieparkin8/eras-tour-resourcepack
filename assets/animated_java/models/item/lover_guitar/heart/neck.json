{"parent":"animated_java:item/lover_guitar/neck","textures":{"2":"minecraft:block/outfits/lover/heart_layer_1"}}