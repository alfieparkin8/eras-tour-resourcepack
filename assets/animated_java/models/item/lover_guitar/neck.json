{"textures":{"1":"minecraft:block/guitars/lover_pink"},"elements":[{"from":[-2.91821985545835,16.152149106124,9.9809813617345],"to":[-1.411943704830735,28.576645112209967,11.00342335488779],"rotation":{"angle":0,"axis":"y","origin":[13.166983643971092,16.09737542791936,13.879041460631418]},"faces":{"north":{"uv":[3,3.25,3.25,5.5],"texture":"#1"},"east":{"uv":[2.5,3.625,2.6875,5.875],"texture":"#1"},"south":{"uv":[3.25,3.25,3.5,5.5],"texture":"#1"},"west":{"uv":[2.75,3.625,2.9375,5.875],"texture":"#1"},"up":{"uv":[6.5,5.5625,6.25,5.375],"texture":"#1"},"down":{"uv":[6.5,5.625,6.25,5.8125],"texture":"#1"}}},{"from":[-3.05515405096995,31.762647394446557,9.9809813617345],"to":[-1.2750095093191334,32,11.00342335488779],"rotation":{"angle":0,"axis":"y","origin":[13.166983643971092,16.09737542791936,13.879041460631418]},"faces":{"north":{"uv":[7.5,2,7.8125,2.0625],"texture":"#1"},"east":{"uv":[7.5,3.5,7.6875,3.5625],"texture":"#1"},"south":{"uv":[7.5,2.125,7.8125,2.1875],"texture":"#1"},"west":{"uv":[3.625,7.5,3.8125,7.5625],"texture":"#1"},"up":{"uv":[6.5625,3.6875,6.25,3.5],"texture":"#1"},"down":{"uv":[3.9375,6.25,3.625,6.4375],"texture":"#1"}}},{"from":[-3.05515405096995,28.576645112209967,9.9809813617345],"to":[-1.2750095093191334,28.78661087866109,11.00342335488779],"rotation":{"angle":0,"axis":"y","origin":[13.166983643971092,16.09737542791936,13.879041460631418]},"faces":{"north":{"uv":[7.5,2.625,7.8125,2.6875],"texture":"#1"},"east":{"uv":[7.5,4.5,7.6875,4.5625],"texture":"#1"},"south":{"uv":[2.75,7.5,3.0625,7.5625],"texture":"#1"},"west":{"uv":[7.5,4.75,7.6875,4.8125],"texture":"#1"},"up":{"uv":[6.5625,4.4375,6.25,4.25],"texture":"#1"},"down":{"uv":[4.9375,6.25,4.625,6.4375],"texture":"#1"}}},{"from":[-1.1380753138075317,30.94104222137695,10.318752377329783],"to":[-0.8642069227843301,31.17839482693039,10.665652339292507],"rotation":{"angle":0,"axis":"y","origin":[13.166983643971092,16.09737542791936,13.879041460631418]},"faces":{"north":{"uv":[7.625,1.875,7.6875,1.9375],"texture":"#1"},"east":{"uv":[2.375,7.625,2.4375,7.6875],"texture":"#1"},"south":{"uv":[2.5,7.625,2.5625,7.6875],"texture":"#1"},"west":{"uv":[2.625,7.625,2.6875,7.6875],"texture":"#1"},"up":{"uv":[2.8125,7.6875,2.75,7.625],"texture":"#1"},"down":{"uv":[2.9375,7.625,2.875,7.6875],"texture":"#1"}}},{"from":[-0.8642069227843301,30.804108025865347,10.318752377329783],"to":[-0.5903385317611267,31.315329022441993,10.665652339292507],"rotation":{"angle":0,"axis":"y","origin":[13.166983643971092,16.09737542791936,13.879041460631418]},"faces":{"north":{"uv":[4.875,7.5,4.9375,7.625],"texture":"#1"},"east":{"uv":[7.5,4.875,7.5625,5],"texture":"#1"},"south":{"uv":[5,7.5,5.0625,7.625],"texture":"#1"},"west":{"uv":[7.5,5,7.5625,5.125],"texture":"#1"},"up":{"uv":[3.0625,7.6875,3,7.625],"texture":"#1"},"down":{"uv":[3.1875,7.625,3.125,7.6875],"texture":"#1"}}},{"from":[-0.8642069227843301,29.84556865728414,10.318752377329783],"to":[-0.5903385317611267,30.356789653860783,10.665652339292507],"rotation":{"angle":0,"axis":"y","origin":[13.166983643971092,16.09737542791936,13.879041460631418]},"faces":{"north":{"uv":[5.125,7.5,5.1875,7.625],"texture":"#1"},"east":{"uv":[7.5,5.125,7.5625,5.25],"texture":"#1"},"south":{"uv":[5.25,7.5,5.3125,7.625],"texture":"#1"},"west":{"uv":[7.5,5.25,7.5625,5.375],"texture":"#1"},"up":{"uv":[4.0625,7.6875,4,7.625],"texture":"#1"},"down":{"uv":[4.6875,7.625,4.625,7.6875],"texture":"#1"}}},{"from":[-1.1380753138075317,29.98250285279574,10.318752377329783],"to":[-0.8642069227843301,30.219855458349183,10.665652339292507],"rotation":{"angle":0,"axis":"y","origin":[13.166983643971092,16.09737542791936,13.879041460631418]},"faces":{"north":{"uv":[3.25,7.625,3.3125,7.6875],"texture":"#1"},"east":{"uv":[3.375,7.625,3.4375,7.6875],"texture":"#1"},"south":{"uv":[3.5,7.625,3.5625,7.6875],"texture":"#1"},"west":{"uv":[3.625,7.625,3.6875,7.6875],"texture":"#1"},"up":{"uv":[3.8125,7.6875,3.75,7.625],"texture":"#1"},"down":{"uv":[3.9375,7.625,3.875,7.6875],"texture":"#1"}}},{"from":[-0.8642069227843301,28.88702928870293,10.318752377329783],"to":[-0.5903385317611267,29.398250285279573,10.665652339292507],"rotation":{"angle":0,"axis":"y","origin":[13.166983643971092,16.09737542791936,13.879041460631418]},"faces":{"north":{"uv":[5.375,7.5,5.4375,7.625],"texture":"#1"},"east":{"uv":[7.5,5.375,7.5625,5.5],"texture":"#1"},"south":{"uv":[5.5,7.5,5.5625,7.625],"texture":"#1"},"west":{"uv":[7.5,5.5,7.5625,5.625],"texture":"#1"},"up":{"uv":[5.1875,7.6875,5.125,7.625],"texture":"#1"},"down":{"uv":[7.6875,5.125,7.625,5.1875],"texture":"#1"}}},{"from":[-1.1380753138075317,29.02396348421453,10.318752377329783],"to":[-0.8642069227843301,29.261316089767973,10.665652339292507],"rotation":{"angle":0,"axis":"y","origin":[13.166983643971092,16.09737542791936,13.879041460631418]},"faces":{"north":{"uv":[7.625,4.625,7.6875,4.6875],"texture":"#1"},"east":{"uv":[4.75,7.625,4.8125,7.6875],"texture":"#1"},"south":{"uv":[4.875,7.625,4.9375,7.6875],"texture":"#1"},"west":{"uv":[7.625,4.875,7.6875,4.9375],"texture":"#1"},"up":{"uv":[5.0625,7.6875,5,7.625],"texture":"#1"},"down":{"uv":[7.6875,5,7.625,5.0625],"texture":"#1"}}},{"from":[-3.7398250285279566,29.84556865728414,10.318752377329783],"to":[-3.465956637504755,30.356789653860783,10.665652339292507],"rotation":{"angle":0,"axis":"y","origin":[13.166983643971092,16.09737542791936,13.879041460631418]},"faces":{"north":{"uv":[6.25,7.5,6.3125,7.625],"texture":"#1"},"east":{"uv":[6.375,7.5,6.4375,7.625],"texture":"#1"},"south":{"uv":[6.5,7.5,6.5625,7.625],"texture":"#1"},"west":{"uv":[6.625,7.5,6.6875,7.625],"texture":"#1"},"up":{"uv":[7.6875,6.6875,7.625,6.625],"texture":"#1"},"down":{"uv":[6.8125,7.625,6.75,7.6875],"texture":"#1"}}},{"from":[-3.465956637504755,29.98250285279574,10.318752377329783],"to":[-3.1920882464815516,30.219855458349183,10.665652339292507],"rotation":{"angle":0,"axis":"y","origin":[13.166983643971092,16.09737542791936,13.879041460631418]},"faces":{"north":{"uv":[6.25,7.625,6.3125,7.6875],"texture":"#1"},"east":{"uv":[7.625,6.25,7.6875,6.3125],"texture":"#1"},"south":{"uv":[6.375,7.625,6.4375,7.6875],"texture":"#1"},"west":{"uv":[7.625,6.375,7.6875,6.4375],"texture":"#1"},"up":{"uv":[6.5625,7.6875,6.5,7.625],"texture":"#1"},"down":{"uv":[6.6875,7.625,6.625,7.6875],"texture":"#1"}}},{"from":[-3.7398250285279566,28.88702928870293,10.318752377329783],"to":[-3.465956637504755,29.398250285279573,10.665652339292507],"rotation":{"angle":0,"axis":"y","origin":[13.166983643971092,16.09737542791936,13.879041460631418]},"faces":{"north":{"uv":[5.875,7.5,5.9375,7.625],"texture":"#1"},"east":{"uv":[6,7.5,6.0625,7.625],"texture":"#1"},"south":{"uv":[6.125,7.5,6.1875,7.625],"texture":"#1"},"west":{"uv":[7.5,6.125,7.5625,6.25],"texture":"#1"},"up":{"uv":[6.1875,7.6875,6.125,7.625],"texture":"#1"},"down":{"uv":[7.6875,6.125,7.625,6.1875],"texture":"#1"}}},{"from":[-3.465956637504755,29.02396348421453,10.318752377329783],"to":[-3.1920882464815516,29.261316089767973,10.665652339292507],"rotation":{"angle":0,"axis":"y","origin":[13.166983643971092,16.09737542791936,13.879041460631418]},"faces":{"north":{"uv":[5.75,7.625,5.8125,7.6875],"texture":"#1"},"east":{"uv":[7.625,5.75,7.6875,5.8125],"texture":"#1"},"south":{"uv":[5.875,7.625,5.9375,7.6875],"texture":"#1"},"west":{"uv":[7.625,5.875,7.6875,5.9375],"texture":"#1"},"up":{"uv":[6.0625,7.6875,6,7.625],"texture":"#1"},"down":{"uv":[7.6875,6,7.625,6.0625],"texture":"#1"}}},{"from":[-3.7398250285279566,30.804108025865347,10.318752377329783],"to":[-3.465956637504755,31.315329022441993,10.665652339292507],"rotation":{"angle":0,"axis":"y","origin":[13.166983643971092,16.09737542791936,13.879041460631418]},"faces":{"north":{"uv":[5.625,7.5,5.6875,7.625],"texture":"#1"},"east":{"uv":[7.5,5.625,7.5625,5.75],"texture":"#1"},"south":{"uv":[5.75,7.5,5.8125,7.625],"texture":"#1"},"west":{"uv":[7.5,5.75,7.5625,5.875],"texture":"#1"},"up":{"uv":[5.6875,7.6875,5.625,7.625],"texture":"#1"},"down":{"uv":[7.6875,5.625,7.625,5.6875],"texture":"#1"}}},{"from":[-3.465956637504755,30.94104222137695,10.318752377329783],"to":[-3.1920882464815516,31.17839482693039,10.665652339292507],"rotation":{"angle":0,"axis":"y","origin":[13.166983643971092,16.09737542791936,13.879041460631418]},"faces":{"north":{"uv":[5.25,7.625,5.3125,7.6875],"texture":"#1"},"east":{"uv":[7.625,5.25,7.6875,5.3125],"texture":"#1"},"south":{"uv":[5.375,7.625,5.4375,7.6875],"texture":"#1"},"west":{"uv":[7.625,5.375,7.6875,5.4375],"texture":"#1"},"up":{"uv":[5.5625,7.6875,5.5,7.625],"texture":"#1"},"down":{"uv":[7.6875,5.5,7.625,5.5625],"texture":"#1"}}},{"from":[-3.1920882464815516,28.78661087866109,9.9809813617345],"to":[-1.1380753138075317,31.762647394446557,11.00342335488779],"rotation":{"angle":0,"axis":"y","origin":[13.166983643971092,16.09737542791936,13.879041460631418]},"faces":{"north":{"uv":[2,4,2.375,4.5625],"texture":"#1"},"east":{"uv":[6,4,6.1875,4.5625],"texture":"#1"},"south":{"uv":[3.5,4,3.875,4.5625],"texture":"#1"},"west":{"uv":[6.125,1.25,6.3125,1.8125],"texture":"#1"},"up":{"uv":[6.625,2.6875,6.25,2.5],"texture":"#1"},"down":{"uv":[6.625,2.75,6.25,2.9375],"texture":"#1"}}},{"from":[-4.150627615062762,10.218333967287943,10.875618105743628],"to":[-0.17953594522632343,11.06732597945987,11.213389121338912],"rotation":{"angle":0,"axis":"y","origin":[8,8,8]},"faces":{"north":{"uv":[3.875,1.5,4.625,1.6875],"texture":"#1"},"east":{"uv":[5,5.5,5.0625,5.6875],"texture":"#1"},"south":{"uv":[5.875,0,6.625,0.1875],"texture":"#1"},"west":{"uv":[5.625,0.5,5.6875,0.6875],"texture":"#1"},"up":{"uv":[7.125,1.0625,6.375,1],"texture":"#1"},"down":{"uv":[7.125,1.75,6.375,1.8125],"texture":"#1"}}},{"from":[-2.7812856599467484,11.03993914035755,10.97603651578547],"to":[-2.7082540890072284,31.07797641688855,11.04906808672499],"rotation":{"angle":0,"axis":"y","origin":[13.240015214910613,18.142259414225943,15.494864967668317]},"faces":{"north":{"uv":[0,4,0.0625,7.6875],"texture":"#1"},"east":{"uv":[0.125,4,0.1875,7.6875],"texture":"#1"},"south":{"uv":[0.25,4,0.3125,7.6875],"texture":"#1"},"west":{"uv":[0.375,4,0.4375,7.6875],"texture":"#1"},"up":{"uv":[7.6875,6.9375,7.625,6.875],"texture":"#1"},"down":{"uv":[7.0625,7.625,7,7.6875],"texture":"#1"}}},{"from":[-1.6127805249144167,11.03993914035755,10.97603651578547],"to":[-1.5488779003423367,31.07797641688855,11.04906808672499],"rotation":{"angle":0,"axis":"y","origin":[13.240015214910613,18.142259414225943,15.494864967668317]},"faces":{"north":{"uv":[0.5,4,0.5625,7.6875],"texture":"#1"},"east":{"uv":[0.625,4,0.6875,7.6875],"texture":"#1"},"south":{"uv":[0.75,4,0.8125,7.6875],"texture":"#1"},"west":{"uv":[0.875,4,0.9375,7.6875],"texture":"#1"},"up":{"uv":[7.6875,7.0625,7.625,7],"texture":"#1"},"down":{"uv":[7.1875,7.625,7.125,7.6875],"texture":"#1"}}},{"from":[-1.8227462913655383,11.03993914035755,10.97603651578547],"to":[-1.7497147204260184,30.411563332065423,11.04906808672499],"rotation":{"angle":0,"axis":"y","origin":[13.240015214910613,18.142259414225943,15.494864967668317]},"faces":{"north":{"uv":[1,4,1.0625,7.5625],"texture":"#1"},"east":{"uv":[1.125,4,1.1875,7.5625],"texture":"#1"},"south":{"uv":[1.25,4,1.3125,7.5625],"texture":"#1"},"west":{"uv":[1.375,4,1.4375,7.5625],"texture":"#1"},"up":{"uv":[7.6875,7.1875,7.625,7.125],"texture":"#1"},"down":{"uv":[7.3125,7.625,7.25,7.6875],"texture":"#1"}}},{"from":[-2.507417268923545,11.03993914035755,10.97603651578547],"to":[-2.434385697984025,30.411563332065423,11.04906808672499],"rotation":{"angle":0,"axis":"y","origin":[13.240015214910613,18.142259414225943,15.494864967668317]},"faces":{"north":{"uv":[1.5,4,1.5625,7.5625],"texture":"#1"},"east":{"uv":[1.625,4,1.6875,7.5625],"texture":"#1"},"south":{"uv":[1.75,4,1.8125,7.5625],"texture":"#1"},"west":{"uv":[1.875,4,1.9375,7.5625],"texture":"#1"},"up":{"uv":[7.6875,7.3125,7.625,7.25],"texture":"#1"},"down":{"uv":[7.4375,7.625,7.375,7.6875],"texture":"#1"}}},{"from":[-2.2974515024724234,11.03993914035755,10.97603651578547],"to":[-2.2335488779003434,29.389121338912133,11.04906808672499],"rotation":{"angle":0,"axis":"y","origin":[13.240015214910613,18.142259414225943,15.494864967668317]},"faces":{"north":{"uv":[2.375,4,2.4375,7.375],"texture":"#1"},"east":{"uv":[2,4.625,2.0625,8],"texture":"#1"},"south":{"uv":[2.125,4.625,2.1875,8],"texture":"#1"},"west":{"uv":[2.25,4.625,2.3125,8],"texture":"#1"},"up":{"uv":[7.6875,7.4375,7.625,7.375],"texture":"#1"},"down":{"uv":[7.5625,7.625,7.5,7.6875],"texture":"#1"}}},{"from":[-2.0235831114492218,11.03993914035755,10.97603651578547],"to":[-1.95968048687714,29.389121338912133,11.04906808672499],"rotation":{"angle":0,"axis":"y","origin":[13.240015214910613,18.142259414225943,15.494864967668317]},"faces":{"north":{"uv":[4.125,4.75,4.1875,8.125],"texture":"#1"},"east":{"uv":[4.25,4.75,4.3125,8.125],"texture":"#1"},"south":{"uv":[4.375,4.75,4.4375,8.125],"texture":"#1"},"west":{"uv":[4.5,4.75,4.5625,8.125],"texture":"#1"},"up":{"uv":[7.6875,7.5625,7.625,7.5],"texture":"#1"},"down":{"uv":[7.6875,7.625,7.625,7.6875],"texture":"#1"}}}],"display":{"head":{"rotation":[0,180,0]}}}