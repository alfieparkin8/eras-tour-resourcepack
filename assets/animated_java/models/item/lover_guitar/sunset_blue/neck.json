{"parent":"animated_java:item/lover_guitar/neck","textures":{"1":"minecraft:block/guitars/lover_blue"}}