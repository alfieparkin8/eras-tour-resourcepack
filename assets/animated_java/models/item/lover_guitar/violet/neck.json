{"parent":"animated_java:item/lover_guitar/neck","textures":{"2":"minecraft:block/outfits/lover/violet_layer_1"}}