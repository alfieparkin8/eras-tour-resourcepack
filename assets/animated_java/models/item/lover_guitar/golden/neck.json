{"parent":"animated_java:item/lover_guitar/neck","textures":{"2":"minecraft:block/outfits/lover/golden_layer_1","3":"minecraft:block/outfits/lover/golden_boots_layer_1"}}